.github/workflows/continuous-integration.yml

```diff
@@ -392,6 +392,16 @@ jobs:
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
           MAVEN_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
           MAVEN_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+      - name: Cancel workflow
+        if: failure()
+        uses: andymckay/cancel-action@0.5
```

Comment:
is this also going to revert the changes from change_version.sh?

Reply:
As change_version.sh doesn't commit anything or upload any artifacts, I see no reason why it wouldn't be reverted. How could we check this for sure?

Reply:
mvn version:commit is not committing anything?

Reply:
As can be seen in our repo history, after a release, we have one automatical commit, for example the "[Updated poms to version 9.1.1-SNAPSHOT](https://github.com/kadai-io/kadai/commit/24090bd351d365d9a2adc207776c7003e1dcba9f)" commit for last update. This commit is done in the next step in ci/commitPoms.sh. The step that I added the cacelation to was previously also there, but produced no additional commit. According to the [doc](https://www.mojohaus.org/versions/versions-maven-plugin/commit-mojo.html), mvn versions:commit doesn't commit anything in the classical sense. However, it still would be better to prove or try out if the changes by the change_version scripts are revertable by the cancel actions.